### PR TITLE
test: updated tests with aghelper table methods

### PR DESCRIPTION
**Pull request by NandanAnantharamu**
updated isSelectedRow() with table_SelectedRow()

/ok-to-test tags="@tag.All"
> [!TIP]
> 🟢 🟢 🟢 All cypress tests have passed! 🎉 🎉 🎉
> Workflow run: <https://github.com/appsmithorg/appsmith/actions/runs/9312024901>
> Commit: c308cbbf4bde7038466a7c93a0e07f12404a9f36
> Cypress dashboard url: <a href="https://internal.appsmith.com/app/cypress-dashboard/rundetails-65890b3c81d7400d08fa9ee5?branch=master&workflowId=9312024901&attempt=1" target="_blank">Click here!</a>

### CODE REVIEW

File: app/client/cypress/e2e/Regression/ClientSide/Binding/TextTableV2_spec.js [hunk 1]
```diff
@@ -35,7 +36,8 @@ describe(
       /**
        * @param(Index)  Provide index value to select the row.
        */
-      cy.isSelectRow(1);
+      table.SelectTableRow(1, 0, true, "v2");
+      // cy.isSelectRow(1);
```
Comment (ApekshaBhosale) [inline app/client/cypress/e2e/Regression/ClientSide/Binding/TextTableV2_spec.js:40]: remove commented code

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/TableV1/Table_spec.js [hunk 2]
```diff
@@ -134,7 +134,8 @@ describe(
 
   it("Table Widget Functionality To Verify The Visiblity mode functionality", function() {
    _.deployMode.NavigateBacktoEditor();
-    cy.isSelectRow(1);
+    _.table.SelectTableRow(1);
+    // cy.isSelectRow(1);
```
Comment (ApekshaBhosale) [inline app/client/cypress/e2e/Regression/ClientSide/Widgets/TableV1/Table_spec.js:138]: do we not need `_.table.SelectTableRow(1);` here?

remove commented code

**Review by ApekshaBhosale — CHANGES_REQUESTED** [review]

File: app/client/cypress/e2e/Regression/ClientSide/Binding/TextTableV2_spec.js (hunk 1 above)
Comment (NandanAnantharamu) [reply app/client/cypress/e2e/Regression/ClientSide/Binding/TextTableV2_spec.js]: done

**Review by ApekshaBhosale — APPROVED** [review]

**Pull request merged**